lib/rules/no-restricted-properties.js

```diff
@@ -155,6 +155,10 @@ module.exports = {
 				const message = matchedObjectProperty.message
 					? ` ${matchedObjectProperty.message}`
 					: "";
+				const allowedPropertiesMessage =
+					matchedObjectProperty.allowProperties
```

Comment:
Can we clean this up a bit?
```suggestion
					? ` Only these properties are allowed:  ${matchedObjectProperty.allowProperties.join(", ")}.`
```